src/main/java/christmas/model/Menu.java

```diff
@@ -0,0 +1,133 @@
+package christmas.model;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Collections;
+import java.util.Objects;
+
+public enum Menu {
+    mushroomSoup("appetizer", "양송이수프", 6000, 0),
+    tapas("appetizer", "타파스", 5500, 0),
+    caesarSalad("appetizer", "시저샐러드", 8800, 0),
+    tBoneSteak("main", "티본스테이크", 55000, 0),
+    barbecuedRibs("main", "바비큐립", 54000, 0),
+    seafoodPasta("main", "해산물파스타", 35000, 0),
+    christmasPasta("main", "크리스마스파스타", 25000, 0),
+    chocoCake("dessert", "초코케이크", 15000, 0),
+    iceCream("dessert", "아이스크림", 5000, 0),
+    zeroCoke("drink", "제로콜라", 3000, 0),
+    redWine("drink", "레드와인", 60000, 0),
+    champagne("drink", "샴페인", 25000, 0);
+
+    private final String menuType;
+    private final String menu;
+    private final int price;
+    private int cnt;
+
+    Menu(String menuType, String menu, int price, int cnt) {
+        this.menuType = menuType;
+        this.menu = menu;
+        this.price = price;
+        this.cnt = cnt;
+    }
+
+    public static List<String> getAllMenu() {
+        List<String> allMenu = new ArrayList<>();
+        for (Menu eachMenu : Menu.values()) {
+            allMenu.add(eachMenu.menu);
+        }
+        return allMenu;
+    }
+
+    public static int countAllCnt() {
+        int allCnt = 0;
+        for (Menu eachMenu : Menu.values()) {
+            allCnt += eachMenu.cnt;
+        }
+        return allCnt;
+    }
+
+    public static void removeAllCnt() {
+        Menu[] menuValue = Menu.values();
+        for (Menu eachMenuValue : menuValue) {
+            if (eachMenuValue.cnt != 0) {
+                eachMenuValue.cnt = 0;
+            }
```

Comment:
이 부분도 Stream을 사용하면 더 깔끔하겠어요!
Menu를 담는 부분도 Array가 아니라 List를 활용해보시는건 어떨까요?
우테코 공통 피드백에 있는 내용을 꼭 참고해보세요!!

```suggestion
        List<Menu> menuValue = Arrays.stream(Menu.values()).toList()

menuValue.stream()
    .forEach(() -> eachMenuValue.cnt = 0);

Reply:
물론 Initialize의 필요성과 목적성보다는, 문법 위주로 리뷰 드리는 내용이니 참고하세요!

Reply:
타입을 변경하다보니 헷갈려 List 사용을 놓치고 갔던 것 같습니다 😱
정말 훨씬 깔끔해지네요 .. 정말 많이 알게됩니다 감사합니다 (_ _)